assets/src/edit-story/components/panels/textStyle/padding.js

```diff
@@ -52,12 +52,12 @@ function PaddingControls({ selectedElements, pushUpdateForObject }) {
     DEFAULT_PADDING
   );
 
-  const [lockPaddingRatio, setLockPaddingRatio] = useState(true);
+  const lockPadding = padding.locked === MULTIPLE_VALUE ? true : padding.locked;
```

Comment:
In this case I actually think we need the default of `false`. Otherwise, one could never even see that one of the paddings is different. For consistency I'm wondering if we need to make aspect ration false by default.

Reply:
hmm... if we want single input in case of lock ratio, it makes sense to make ti false when there are multiple values available. But I think default ratio it true by default? @pbakaus

Reply:
padding common lock ratio updated to false when multiple lock ratio combined. And added comment on padding and size panel.